src/stable/test/index.ts

```diff
@@ -46,14 +54,81 @@ export function runTests(
         });
     }
 
-    if (shouldRecordBenchmarks === true && canisterNames !== undefined) {
-        const canisterNamesArray = Array.isArray(canisterNames)
-            ? canisterNames
-            : [canisterNames];
+    // TODO is there a better way to get the canister name?
```

Comment:
Do these need to be resolved still? I see the get dfx.json function down there

Reply:
I just need to delete the comments